module-check/src/main/clojure/clojure/core/typed/base_env.clj

```diff
@@ -740,6 +740,9 @@ clojure.string/trim [String -> String]
 clojure.string/trimr [String -> String]
 clojure.string/triml [String -> String]
 
+clojure.template/apply-template [ (t/Vec t/Any ) t/Any (t/List t/Any ) -> t/Any ]
+clojure.template/do-template [ (t/Vec t/Any ) t/Any (t/AnyValue * ) -> t/Any ]
```

Comment:
Ok, I will make these changes. I will also remove the tests and annotations for do-template